src/FakeItEasy.Analyzer/Resources.resx

```diff
@@ -117,6 +117,18 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="NonVirtualSetupDescription" xml:space="preserve">
+    <value>Non virtual members can not be intercepted.</value>
+    <comment>An optional longer localizable description of the diagnostic.</comment>

```

Comment:
I'm not an expert, but should this message be here? I'd've assumed either a longer description, or the lack of the `comment` tag.